src/demo/models/guide.js

```diff
@@ -4,53 +4,86 @@ import {AppMode, Modes} from '../constants';
 const guides = [
   {
     id: 'fishvtrash-training-init',
-    text: "Let's clean up the ocean!",
+    text: "Garbage dumped in the ocean and rivers affects water health and marine life.  In this activity you will program an artificial intelligence (AI) to identify trash.  Let's clean up the ocean!",
```

Comment:
Should we add image support so that we can illustrate these points?  I can imagine a thumbnail image to the left of the text here...